web/pgadmin/tools/sqleditor/static/js/components/sections/ResultSetToolbar.jsx

```diff
@@ -167,10 +182,59 @@ export function ResultSetToolbar({canEdit, totalRowCount}) {
       }
     },
   ], queryToolCtx.mainContainerRef);
+  function setEditorPosition(cellEle, editorEle) {
```

Comment:
You have copy pasted this function. Don't duplicate code.

Reply:
As you can see the closest element that we need in both the cases are different and the value that we are adding to get the top in position is also different, Do I need to reuse the function anyhow?

<img width="1306" alt="Screenshot 2024-05-22 at 5 34 51 PM" src="https://github.com/pgadmin-org/pgadmin4/assets/122535205/c956e09c-04ed-40fe-b726-29af568b894f">

Reply:
For a single word you copied the complete function? Great!

Reply:
Okay, I will reuse the function.